AppDashboard/lib/app_dashboard_data.py

```diff
@@ -170,6 +172,75 @@ def __init__(self, helper=None):
     """
     self.helper = helper or AppDashboardHelper()
 
+  def build_dict(self):
+    LOOKUP_DICT = {}
+    LOOKUP_DICT['cloud_stats'] = {"title":"Cloud Status",
+                                  "link":"/status/cloud",
+                                  "template":"status/cloud.html"}
+    LOOKUP_DICT['database_stats'] = {"title":"Database",
+                                  "template":"apps/database.html"}
+    LOOKUP_DICT['memcache_stats'] = {"title":"Memcache",
+                                     "template":"apps/memcache.html"}
+    LOOKUP_DICT['upload_app'] = {"title":"Upload App",
+                                 "link":"/apps/new",
+                                 "template":"apps/new.html"}
+    LOOKUP_DICT['delete_app'] = {"title":"Delete App",
+                                 "link":"/apps/delete",
+                                 "template":"apps/delete.html"}
+    LOOKUP_DICT['manage_users'] = {"title":"Manage Users",
+                                   "link":"/authorize",
+                                   "template":"authorize/cloud.html"}
+    LOOKUP_DICT['logging'] = {"title":"Logs",
+                              "link":"/logs",
+                              "template":"logs/main.html"}
+    LOOKUP_DICT['monit'] = {"title":"Monit",
+                             "link":self.get_monit_url()}
+    LOOKUP_DICT['taskqueue'] = {"title":"TaskQueue",
+                             "link":self.get_flower_url()}
+
+    LOOKUP_DICT['app_management'] = [self.helper.can_upload_apps(),
```

Comment:
Does this key need to be included in the return dict at all if `self.helper.can_upload_apps()` evaluates to `False`?
It would make much more sense to me if it looked like
```
if self.helper.can_upload_apps():
  LOOKUP_DICT['app_management'] = {'title': 'App Mangement' ...}
```